src/DependencyInjection/Configuration.php

```diff
@@ -21,10 +21,20 @@ public function getConfigTreeBuilder()
         $rootNode = $treeBuilder->root('hub_spot');
 
         $rootNode->children()
-            ->scalarNode('api_key')
-                ->isRequired()
+            ->scalarNode('key')
             ->end()
-        ->end();
+            ->arrayNode('proxy')
+                ->children()
+                    ->scalarNode('custom_url')
```

Comment:
```suggestion
                    ->scalarNode('url')
```
what's the need for `custom_`?